go/libkb/id_table.go

```diff
@@ -453,6 +453,19 @@ func (l *TrackChainLink) GetTrackedKeys() ([]TrackedKey, error) {
 
 	var res []TrackedKey
 
+	// If the eldest kid in this tracking statement is a PGP key, it's
+	// implicitly tracked, in addition to the "pgp_keys" section that follows.
+	eldestKeyJSON := l.payloadJSON.AtPath("body.track.key")
+	eldestTrackedKey, err := trackedKeyFromJSON(eldestKeyJSON)
+	if err != nil {
+		return nil, err
+	}
+	var emptyPGPFingerprint PGPFingerprint
+	if !eldestTrackedKey.Fingerprint.Eq(emptyPGPFingerprint) {
```

Comment:
Inside the `trackedKeyFromJSON` function we can see that the Fingerprint field is missing, but the only indication after we leave that function is that the returned Fingerprint is all zeroes. (It's not a pointer, so it can't be nil.) I guess the other option would be to have that function return a second value?